templates/CRM/Financial/Page/BatchTransaction.tpl

```diff
@@ -140,7 +140,7 @@ function saveRecord(recordID, op, recordBAO, entityID) {
     window.location.href = CRM.url('civicrm/financial/batch/export', {reset: 1, id: recordID, status: 1});
     return;
   }
-  var postUrl = {/literal}"{crmURL p='civicrm/ajax/rest' h=0 q="className=CRM_Financial_Page_AJAX&fnName=assignRemove&qfKey={$financialAJAXQFKey}"}"{literal};
+  var postUrl = {/literal}"{$financialUrl}"{literal};
```

Comment:
Two things here:

1. In my testing, the `assign()`ed value doesn't actually show up here. So I just get `postUrl = "";`
2. The `{crmURL}` call has `h=0` (no HTML encoding). The new call above uses `CRM_Utils_System::url()` which would give HTML encoding. (FWIW, `Civi::url('civicrm/ajax/rest)` would do the sane thing and default to canonical text...*)

See replacement: #31738